io/io/src/TFileMerger.cxx

```diff
@@ -776,8 +776,10 @@ Bool_t TFileMerger::MergeRecursive(TDirectory *target, TList *sourcelist, Int_t
                   TDirectory *ndir = nextsource->GetDirectory(dirpath);
                   // For consistency (and persformance), we reset the MustCleanup be also for those
                   // 'key' retrieved indirectly.
-                  ndir->ResetBit(kMustCleanup);
-                  delete ndir;
+                  if (ndir) {
+		    ndir->ResetBit(kMustCleanup);
+		    delete ndir;
+		  }
```

Comment:
Please remove the TABs in these three lines.

Reply:
Done.